constrained_ik/constrained_ik_constraint_plugin_description.xml

```diff
@@ -1,4 +1,4 @@
-<library path="lib/libconstrained_ik_constraints">
+<library path="libconstrained_ik_constraints">
```

Comment:
Just wanted to check, is this a new change in Kinetic or has this always been wrong?